tutorials/setup.md

```diff
@@ -17,28 +17,80 @@ Latest version of `runc` is expected to be installed on the system. It is picked
 
 **Required**
 
-Fedora, CentOS, RHEL, and related distributions:
+Fedora, RHEL<=7, CentOS and related distributions:
 
 ```bash
 yum install -y \
-  btrfs-progs-devel \
-  containers-common \
-  device-mapper-devel \
-  git \
-  glib2-devel \
-  glibc-devel \
-  glibc-static \
-  go \
-  golang-github-cpuguy83-go-md2man \
-  gpgme-devel \
-  libassuan-devel \
-  libgpg-error-devel \
-  libseccomp-devel \
-  libselinux-devel \
-  pkgconfig \
-  runc
+btrfs-progs-devel \
+containers-common \
+device-mapper-devel \
+git \
+glib2-devel \
+glibc-devel \
+glibc-static \
+go \
+gpgme-devel \
+libassuan-devel \
+libgpg-error-devel \
+libseccomp-devel \
+libselinux-devel \
+pkgconfig \
+runc
 ```
 
+
+RHEL 8 distributions:
+
+Make sure you are subscribed to the following repositories: \
+BaseOS/x86_64 \
+Appstream/x86_64
+
+Follow the guide below to subscribe to the repositories if not already subscribed: \
+https://access.redhat.com/solutions/265523
+
+
+This requires go version 1.12 or greater. Use the link below to install the appropriate go version for the system:\
+(Replace apt-get with yum for RHEL based machines)
+
+https://tecadmin.net/install-go-on-ubuntu/
```

Comment:
We don't need this link. The official docs cover this well.

Reply:
Sure, I will remove that link.